app/editor/src/features/admin/work-orders/WorkOrderForm.tsx

```diff
@@ -83,24 +84,27 @@ export const WorkOrderForm: React.FC = () => {
                   options={userOptions}
                   name="requestorId"
                   label="Requestor"
+                  isDisabled={!isAdmin}
                   value={userOptions.find((s) => s.value === values.requestorId) || ''}
                 />
-                <FormikTextArea name="description" label="Description" />
+                <FormikTextArea name="description" label="Description" disabled={!isAdmin} />
```

Comment:
Not sure there is any value in allowing non-admins access to the page if they can't do anything.

Reply:
I think the ticket said allow editors to view, only reason I added that in!